sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuBoundAttribute.scala

```diff
@@ -68,7 +68,7 @@ object GpuBindReferences extends Logging {
   def bindGpuReferences[A <: Expression](
       expressions: Seq[A],
       input: AttributeSeq): Seq[GpuExpression] =
-    expressions.map(GpuBindReferences.bindGpuReference(_, input))
+    expressions.map(GpuBindReferences.bindGpuReference(_, input)).toList
```

Comment:
Curious why `toList` was used instead of `toArray` as I've seen done elsewhere to force a manifest?  Either way, probably worth adding a comment so someone doesn't "optimize" it later. :smile:

Reply:
added a comment https://github.com/NVIDIA/spark-rapids/pull/2120/files#diff-7718c93b5e463c82611c693aa760c934343e3d8047dd800f1d0c00e9ab8afa52R63

Reply:
The comment isn't anywhere near this code.  Would be nice to have a simple one-liner like `// Force list to avoid recursive Java serialization of Seq` so maintainers don't think the `toList` is superfluous.